datatypes.go

```diff
@@ -42,16 +42,16 @@ const (
 
 // NotificationEvent represents trigger state changes event
 type NotificationEvent struct {
-	IsTriggerEvent   bool               `json:"trigger_event,omitempty"`
-	Timestamp        int64              `json:"timestamp"`
-	Metric           string             `json:"metric"`
-	Value            *float64           `json:"value,omitempty"`
+	IsTriggerEvent   bool               `json:"trigger_event,omitempty" example:"true"`
+	Timestamp        int64              `json:"timestamp" example:"1590741878"`
+	Metric           string             `json:"metric" example:"carbon.agents.*.metricsReceived"`
```

Comment:
Can this field actually contain a `*` character?

Reply:
This strange field used to be used before when sending sms before I fixed the problem